Documentation/git-config.txt

```diff
@@ -235,8 +235,12 @@ FILES
 If not set explicitly with `--file`, there are four files where
 'git config' will search for configuration options:
 
+$PROGRAMDATA/Git/config::
+	System-wide configuration file for *all* Git-related software running
```

Comment:
Please note that this is Windows-only, too, so starting the description with `(Windows-only)` may be a very good idea here.

Together with @nalla's insightful observation, maybe something like this?

> (Windows-only): System-wide configuration file shared with other Git implementations.